eventOrganizer/Date.java

```diff
@@ -1,7 +1,15 @@
 package eventOrganizer;
-import java.util.Calendar;
 
+
+/**
+ * @Author Seifeldeen Mohamed
```

Comment:
this is backwards, looking at the document for an example:
/**
First, a single, very descriptive sentence describing the class.
Then, additional lines of description to elaborate the details if necessary.
@author studentName1, studentName2
*/
public class Student {